splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1815,6 +1815,14 @@
               "os"
             ]
           }
+        },
+        "supportedThemes": {
```

Comment:
Let's add this to the `init-template` in templates folder. I believe newly initialized add-on should be compatible with dark theme.

Reply:
Done